src/Summoner/Tui.hs

```diff
@@ -173,6 +179,10 @@ app = App
         VtyEvent V.EvResize {}       -> continue s
         -- TODO: should we cancel on Esc and apply on Ctrl+Enter?
         VtyEvent (V.EvKey V.KEsc []) -> halt s
+        MouseDown n _ _ _ -> case n of
+            GitHubEnable  -> handleFormEvent ev s >>= continue . mkForm . formState
+            GitHubDisable -> handleFormEvent ev s >>= continue . mkForm . formState
+            _             -> handleFormEvent ev s >>= continue
```

Comment:
`ev`?

Reply:
`ev` is for event. This probably will be refactored later.

Reply:
`event` if nobody uses it/